.github/workflows/sourcehold.yml

```diff
@@ -0,0 +1,50 @@
+name: Sourcehold
+
+on: [push]
```

Comment:
Maybe also add on pull request, so we can see whether PR's actually build!

Reply:
It does it automatically on PRs too - see my test PR here: https://github.com/atmaxinger/Sourcehold/pull/2

Reply:
Nevermind, explicitly stating it doesn't harm anyone. Done.